main/src/ca/uwaterloo/flix/language/phase/Regions.scala

```diff
@@ -378,13 +378,16 @@ object Regions {
       * where `trueVars` are the variables ascribed the value TRUE,
       * and all other variables are ascribed the value FALSE.
       */
-    def eval(tpe: Type, trueVars: SortedSet[Type.Var]): Boolean = tpe match {
```

Comment:
This function was improperly updated to set effects. It crashes for concrete effects and difference and xor. I've fixed this.

This is observable if you have `Abort` across a region scope fx